src/DotNetWorker/DotNetWorker.csproj

```diff
@@ -8,7 +8,7 @@
     <AssemblyName>Microsoft.Azure.Functions.Worker</AssemblyName>
     <RootNamespace>Microsoft.Azure.Functions.Worker</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <MinorProductVersion>16</MinorProductVersion>
+    <MinorProductVersion>17</MinorProductVersion>
```

Comment:
Is it okay if we include the deferred binding changes in the .17 release or does this need to go out asap?

Reply:
I take this back, we can do deferred binding in 18